.travis.yml

```diff
@@ -6,12 +6,7 @@ dart:
 jobs:
   include:
     - stage: analyze_and_format
-      name: "Analyze lib/ (no experiment flag)"
```

Comment:
FYI: you can go back to the old `dart_task` syntax – a lot less verbose

Reply:
ack